externals/tinyxml/tinyxml2.cpp

```diff
@@ -1962,6 +1962,7 @@ XMLDocument::XMLDocument( bool processEntities, Whitespace whitespace ) :
     _errorID(XML_SUCCESS),
     _whitespace( whitespace ),
     _charBuffer( 0 )
+    // Not initialized _errorLineNum, _parseCurLineNum

```

Comment:
please don't fix stuff in our "externals", these files are just copies. the next time we copy the tinyxml2 files these fixes would be lost..

feel free to fix this upstream. I believe this is: http://github.com/leethomason/tinyxml2